bin/newman.js

```diff
@@ -22,7 +22,7 @@ program
     .usage('<collection> [options]')
     .option('-e, --environment <path>', 'Specify a URL or path to a Postman Environment')
     .option('-g, --globals <path>', 'Specify a URL or path to a file containing Postman Globals')
-    .option('-r, --reporters [reporters]', 'Specify the reporters to use for this run', util.cast.csvParse, ['cli'])
+    .option('-r, --reporters [reporters...]', 'Specify the reporters to use for this run', util.cast.reporterParse, ['cli'])
```

Comment:
I think the default `['cli']`  option can be removed as we're returning that value from the cast util.

Reply:
This is present as a default if someone does not provide any arguments after the ```-r``` command.

Reply:
Oh, I see. Thanks for the clarification.